### DISCUSSION TITLE
芜，API被封咯

### DISCUSSION THREAD

**Discussion by PrussianBlueCN** [ikechan8370/chatgpt-plugin#230] [General]
不太清楚是因为沙雕群友聊了太过逆天的东西还是因为被检测到长期在香港发起请求，反正就突然炸掉了

**Comment by ikechan8370** [comment]
聊天内容不会导致被封的。一般是账号来源的问题？

**Reply by PrussianBlueCN** [reply]
自己注册的账号，除了手机用的sms-activate其他的都是自己的，暂时不太清楚具体原因。而且本地打开网页端仍然能用的，只有API寄了就很奇怪。打算晚点美国佬上班了问问客服，如果有有价值信息会发在这

**Comment by PrussianBlueCN** [comment]
更新：用ChatGPT写了一封申诉信过去，没有回信，今天发现整个账号被端了
